mlfinlab/microstructural_features/first_generation.py

```diff
@@ -0,0 +1,112 @@
+"""
+First generation features (Roll Measure/Impact, Corwin-Schultz spread estimator)
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def get_roll_measure(close_prices: pd.Series, window: int = 20) -> pd.Series:
+    """
+    Get Roll Measure
```

Comment:
Lets add better docstrings. Explaining what the roll measure is. What part of the book or paper it comes from. 

This docstrings form part of documentation.